src/registry.js

```diff
@@ -399,12 +399,15 @@ class AtlasRegistry {
    *   by our registered meters.
    */
   measurements() {
-    let a = [];
+    let a = new Array(this.metersMap.size * 4);
+    let len = 0;
 
     for (let meter of this.metersMap.values()) {
-      Array.prototype.push.apply(a, meter.measure());
+      const measures = meter.measure();
+      len = len + measures.length;
+      Array.prototype.push.apply(a, measures);
     }
-    return a;
+    return Array.prototype.slice.call(a, 0, len);
```

Comment:
It would be nice to add a test, I'm pretty sure this would just return an array that is all `undefined` since the first part of the array that was allocated would not get filled.

Reply:
I double checked and the `measurements` test which exists for the method already covers this change.

Reply:
Ok, good to know. I didn't see a build failure, but just double checked and this project still uses TravisCI and I think PR builds are disabled. We'll need to switch it to GitHub Actions so our PRs will have the tests checked.